src/client/JoinPrivateLobbyModal.ts

```
@@ -170,11 +247,104 @@ export class JoinPrivateLobbyModal extends LitElement {
   private async joinLobby(): Promise<void> {
     const lobbyId = this.lobbyIdInput.value;
-    console.log(`Joining lobby with ID: ${lobbyId}`);
+    console.log(`Checking lobby with ID: ${lobbyId}`);
     this.message = `${translateText("private_lobby.checking")}`;
 
     try {
-      // First, check if the game exists in active lobbies
+      // First, check if the lobby is deployed on-chain
+      console.log("Checking if lobby is deployed on-chain...");
+      const isOnChain = await isLobbyOnChain(lobbyId);
+      
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)